minimal.env.json

```diff
@@ -8,5 +8,9 @@
 
   "========== Service Curation settings ==========": "",
   "CURATION_GITHUB_REPO": "sample-curated-data",
-  "CURATION_GITHUB_TOKEN": "<GitHub token here>"
+  "CURATION_GITHUB_TOKEN": "<GitHub token here>",
+
+  "========== Heapstats Logging settings (OPTIONAL) ==========": "",
+  "LOG_NODE_HEAPSTATS": "<true|false>",
+  "LOG_NODE_HEAPSTATS_INTERVAL_MS": "<time_in_milliseconds (e.g. '30000')>"
```

Comment:
As a separate PR, please add these to our [master document](https://github.com/clearlydefined/operations/blob/main/overview.md) of environment variables.

Reply:
Yep, will do!  Thank you for the link to this

Reply:
Created the following PR to update the master document:
https://github.com/clearlydefined/operations/pull/117